src/views/EmergencyDetails/FieldReportStats/index.tsx

```diff
@@ -0,0 +1,293 @@
+import { useContext } from 'react';
+import { generatePath } from 'react-router-dom';
+
+import Container from '#components/Container';
+import DateOutput from '#components/DateOutput';
+import Link from '#components/Link';
+import RouteContext from '#contexts/route';
+import TextOutput from '#components/TextOutput';
+import useTranslation from '#hooks/useTranslation';
+import { resolveToComponent } from '#utils/translation';
+import type { GoApiResponse } from '#utils/restRequest';
+import {
+    FIELD_REPORT_STATUS_EARLY_WARNING,
+    DISASTER_TYPE_EPIDEMIC,
+} from '#utils/constants';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type EventItem = GoApiResponse<'/api/v2/event/{id}'>;
+type FieldReport = EventItem['field_reports'][number];
+
+interface Props {
+    disasterType: EventItem['dtype'];
+    report: FieldReport;
+}
+
+function FieldReportStats(props: Props) {
+    const {
+        disasterType,
+        report,
+    } = props;
+
+    const {
+        fieldReportDetails: fieldReportDetailsRoute,
+    } = useContext(RouteContext);
+
+    const strings = useTranslation(i18n);
+
+    const reportLink = resolveToComponent(
+        strings.source,
+        {
+            link: (
+                <Link
+                    to={generatePath(
+                        fieldReportDetailsRoute.absolutePath,
+                        { fieldReportId: report.id },
+                    )}
+                >
+                    {report.summary}
+                </Link>
+            ),
+            date: (
+                <DateOutput value={report.updated_at ?? report.created_at} />
+            ),
+        },
+    );
+
+    const numAffected = report.num_affected
+        ?? report.gov_num_affected ?? report.other_num_affected;
+    const numInjured = report.num_injured
+        ?? report.gov_num_injured ?? report.other_num_injured;
+    const numDead = report.num_dead
+        ?? report.gov_num_dead ?? report.other_num_dead;
+    const numMissing = report.num_missing
+        ?? report.gov_num_missing ?? report.other_num_missing;
+    const numDisplaced = report.num_displaced
+        ?? report.gov_num_displaced ?? report.other_num_displaced;
+    const numAssisted = report.num_assisted
+        ?? report.gov_num_assisted ?? report.other_num_assisted;
+
+    if (report.status === FIELD_REPORT_STATUS_EARLY_WARNING) {
```

Comment:
Let's use reportType insetad of report.status, report.is_covid_report and disasterType